kolena/workflow/evaluator.py

```diff
@@ -337,13 +389,13 @@ def display_name(self) -> str:
 
 class Evaluator(metaclass=ABCMeta):
     """
-    An :class:`kolena.workflow.Evaluator` transforms inferences into metrics.
+    An `Evaluator` transforms inferences into metrics.
 
-    Metrics are computed at the individual test sample level (:class:`kolena.workflow.MetricsTestSample`), in aggregate
-    at the test case level (:class:`kolena.workflow.MetricsTestCase`), and across populations at the test suite level
-    (:class:`kolena.workflow.MetricsTestSuite`).
+    Metrics are computed at the individual test sample level ([`MetricsTestSample`][kolena.workflow.MetricsTestSample]),
+    in aggregate at the test case level ([`MetricsTestCase`][kolena.workflow.MetricsTestCase]), and across populations
+    at the test suite level ([`MetricsTestSuite`][kolena.workflow.MetricsTestSuite]).
 
-    Test-case-level plots (:class:`kolena.workflow.Plot`) may also be computed.
+    Test-case-level plots ([`Plot`][kolena.workflow.Plot]) may also be computed.
     """
 
     configurations: List[EvaluatorConfiguration]
```

Comment:
For class `Evaluator Configuration` and `Evaluator`, `display_name()` is missing in the new API doc.
Also, input arg `configurations` info is missing?